<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>i cant do this anymore;(((</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="letter" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>861</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border: 1px solid black</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>540</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>XY Projection</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>570</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>XZ Projection</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>600</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>ZY Projection</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply Scaling</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>620</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply Transfer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>540</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>rotation x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>570</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>rotation y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>600</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>rotation z</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>630</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>x scale:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>y scale:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>z scale:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>540</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>x transfer:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>570</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>y transfer:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>600</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>z transfer:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>540</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>x rotation angle:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>570</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>y rotation angle: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>600</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>z rotation angle:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="xScaleSpinBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>540</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <double>0.100000000000000</double>
    </property>
    <property name="maximum">
     <double>6.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="yScaleSpinBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>570</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <double>0.100000000000000</double>
    </property>
    <property name="maximum">
     <double>6.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="zScaleSpinBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>600</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <double>0.100000000000000</double>
    </property>
    <property name="maximum">
     <double>6.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="xTransferSpinBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>540</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <double>-10.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="yTransferSpinBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>570</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <double>-10.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="zTransferSpinBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>600</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <double>-10.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="xRotationSpinBox">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>540</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="yRotationSpinBox">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>570</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="zRotationSpinBox">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>600</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
   </widget>
   <widget class="QLabel" name="outputLabel">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>540</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>897</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>letter</class>
   <extends>QWidget</extends>
   <header>letter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
